snuba/query/timeseries_extension.py

```diff
@@ -122,3 +122,48 @@ def __init__(
             ),
             processor=TimeSeriesExtensionProcessor(timestamp_column),
         )
+
+
+class SessionsTimeSeriesExtensionProcessor(TimeSeriesExtensionProcessor):
```

Comment:
Again extensions are deprecated, so you'll need to subclass `TimeSeriesProcessor` (the query processor) and move this logic there.

Reply:
Thanks for letting me know